pkg/controller/migassetcollection/validation.go

```diff
@@ -28,39 +28,48 @@ const (
 )
 
 // Validate the asset collection resource.
-// Returns error and the total error conditions set.
-func (r ReconcileMigAssetCollection) validate(assetCollection *migapi.MigAssetCollection) (int, error) {
-	totalSet := 0
+func (r ReconcileMigAssetCollection) validate(assetCollection *migapi.MigAssetCollection) error {
+	assetCollection.Status.BeginStagedConditions()
```

Comment:
Thoughts on `EmptyStagedConditions()`? Is this <<< an accurate depiction of what `BeginStagedConditions()` does?

Reply:
This used to be `DeleteUnstagedConditions()` but since it also changes the _mode_, I thought this was clearer.  Not thinking that `Empty` fits.

Reply:
Ah I remember `DeleteUnstaged`. Yeah empty doesn't seem like a good fit knowing that.